src/model.jl

```diff
@@ -1,43 +1,48 @@
-export fit_profile
+export find_solution, find_search_interval
 
 """
-    fit_profile(profile_values::Vector{Float64},
-                target_mean::Float64)
+(a, b) = find_search_interval(f)
```

Comment:
```suggestion
    (a, b) = find_search_interval(f)
```
The 4 spaces are equivalent to wrapping into triple-backticks.